src/Symfony/Bundle/FrameworkBundle/EventListener/SessionListener.php

```diff
@@ -11,17 +11,32 @@
 
 namespace Symfony\Bundle\FrameworkBundle\EventListener;
 
-use Symfony\Component\HttpKernel\EventListener\SessionListener as BaseSessionListener;
+use Symfony\Component\DependencyInjection\ContainerInterface;
 
-@trigger_error(sprintf('The %s class is deprecated since version 3.3 and will be removed in 4.0. Use %s instead.', SessionListener::class, BaseSessionListener::class), E_USER_DEPRECATED);
+@trigger_error(sprintf('The %s class is deprecated since version 3.3 and will be removed in 4.0. Use Symfony\Component\HttpKernel\EventListener\SessionListener instead.', SessionListener::class), E_USER_DEPRECATED);
 
 /**
  * Sets the session in the request.
  *
  * @author Fabien Potencier <fabien@symfony.com>
  *
- * @deprecated since version 3.3, to be removed in 4.0. Use {@link BaseSessionListener} instead
+ * @deprecated since version 3.3, to be removed in 4.0. Use Symfony\Component\HttpKernel\EventListener\SessionListener instead
  */
-class SessionListener extends BaseSessionListener
+class SessionListener
```

Comment:
not extending anything is wrong, as this misses all the logic which was coming from the parent class in 3.2. Your class here is just broken.